eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/rest/BuildingResource.java

```diff
@@ -62,4 +64,10 @@
     buildingService.deleteBuilding(id);
     return ResponseEntity.noContent().build();
   }
+
+  @PostMapping("/groupdelete")
```

Comment:
DeleteMapping legyen, és nevezzük át az end-pointot megfelelőbbre.

Reply:
deleteMappingre javitva endpoint is javitva illetve mivel PathVariable lett ezért az endpointban az id is szerepel